guides/manual-guide-to-rings.md

```diff
@@ -0,0 +1,323 @@
+# A manual guide to utilizing rings
+
+### Introduction
```

Comment:
Maybe decrement all the headings by 1. We don't have any `h2`s

Reply:
fixed